schemas/v3/measure_v3.xsd

```diff
@@ -145,11 +145,6 @@
 									<xs:documentation>maximum value</xs:documentation>
 								</xs:annotation>
 							</xs:element>
-							<xs:element name="model_dependent" type="xs:boolean" default="false">
-								<xs:annotation>
-									<xs:documentation>true if any of this argument's properties depend on the specific model used.  default_value, choice_values, min_value, and max_value will not be populated for dynamic arguments</xs:documentation>
-								</xs:annotation>
-							</xs:element>
```

Comment:
model_dependent is already defined on line 103: https://github.com/NREL/bcl-gem/pull/27/files#diff-abef9e2be44d09120acb9196ccf8cf361c45e04cb63bf8b4107c225ece4817edR103-R107